src/md/release.md

```diff
@@ -193,7 +193,14 @@ git checkout release-3.2
 dv=3.2.0-incubating-SNAPSHOT
 rv=3.2.1-incubating-SNAPSHOT
 for a in `git grep -l "${dv}"`; do echo $a; sed -i 's/'"${dv}"'/'"${rv}"'/g' $a; done
+```
+Upgrade base version for semantic versioning maven plugin to the published release (X.Y.Z). If there are new artifacts published to maven repositories
```

Comment:
Once release repository is promoted, how long it may take for release artifacts to be visible?